zazu/git_helper.py

```diff
@@ -76,15 +77,24 @@ def install_git_hook(hooks_folder, hook_name, hook_resource_path):
         shutil.copy(hook_resource_path, hook_path)
 
 
+def parse_branch_return_list(branches):
+    ret = branches
+    for i, branch in enumerate(ret):
+        if branch.startswith('* '):
+            ret[i] = branch[2:]
+            break
+    return ret
```

Comment:
We could use something like [inquirer](https://pypi.python.org/pypi/inquirer) to present a menu selection of branches. Possible idea for the future.